src/algebra/category/FinVect.lean

```diff
@@ -31,12 +31,12 @@ universes u
 variables (K : Type u) [field K]
 
 /-- Define `FinVect` as the subtype of `Module.{u} K` of finite dimensional vector spaces. -/
-@[derive [category, λ α, has_coe_to_sort α (Sort*)]]
+@[derive [large_category, λ α, has_coe_to_sort α (Sort*), concrete_category]]
```

Comment:
At some point we should rename this to `FinMod`. Because `finite_dimensional` is defeq to `module.finite K V`.